python/cudf/cudf/tests/utils.py

```diff
@@ -115,7 +115,13 @@ def assert_neq(left, right, **kwargs):
         raise AssertionError
 
 
-def assert_exception(reference_func, actual_func, compare_error_message=True):
+def assert_exceptions_equal(
+    reference_func,
+    actual_func,
+    reference_func_args=None,
+    actual_func_args=None,
```

Comment:
I think we either need to split this up to handle both positional and keyword args, or make these something like a length 2 tuple where the 0th element is a list-like of positional args, and the 1st element is a dict-like of kwargs.

Reply:
Made the changes to this API with tuple input where 1st position will hold args and second holds kwargs 👍